Gemfile

```diff
@@ -127,6 +127,7 @@ group :development, :test do
   gem "rspec-retry", "~> 0.6"
   gem "rubocop", "~> 0.63", require: false
   gem "rubocop-rspec", "~> 1.31"
+  gem "erb_lint", require: false
```

Comment:
Can you limit the gem version to `~> 0.0` and keep this section alphabetical?

Reply:
Fixed that 👌